_posts/2000-01-20-who.md

```diff
@@ -19,7 +19,7 @@ Friends who contributed to Mockito (apologize if I missed somebody):
 **Pascal Schumacher**, **Christian Schwartz**, **Igor Czechowski**, **Patric Fornasier**, **Jim Barritt**,
 **Felix Leipold**, **Liz Keogh**, **Dan North**, **Bartosz Bańkowski**, **David Wallace**.
 
-[Travis CI](https://travis-ci.org/mockito/mockito) and [Bintray](https://bintray.com/mockito/maven/mockito) are used to facilitate continuous delivery.
+[Travis CI](https://travis-ci.org/mockito/mockito) and Bintray are used to facilitate continuous delivery.
 Both tools are great. Thank you Travis, thank you Bintray!
```

Comment:
```suggestion
[GitHub actions](https://github.com/mockito/mockito/actions) and [Shipkit](http://shipkit.org) are used to facilitate continuous delivery.
Both tools are great. Thank you GitHub, thank you Shipkit!
```

Reply:
thank you for checking quickly and give me a suggestion change :bow: